### ISSUE TITLE
Kill confirmation

### ISSUE THREAD

**Issue by Fransje** [inexor-game/code#91]
In my opinion it is important to know exactly when you've killed your opponent. This way you don't have to waste ammo, and you can reorientate quickly on what you want to achieve next.
After having played with Community Edition and the frag message that is implemented there, I came back to the vanilla client, and had difficulties with determining whether or not I had killed my opponent. I think newcomers have this problem too (if they play anything else than insta), for them the game is quite confusing already. They would profit from some kill confirmation too.

Because of these reasons I'd like to suggest two ways of confirming the kill:
    - A visual confirmation. IMHO a frag message is not necessary; if you want to know who you killed,     look at the left upper corner. But for example a red circle around the crosshair that blinks a few times would be great. Not too big and annoying though.
    - A kill sound. Basically just play this sound instead of the hit sound when you give your enemy the fatal blow. Again: shouldn't be annoying, just as subtle as the hit sound is.

And of course vars to toggle both things on/off, but have them toggled on by default (right now the hitsound is off by default I think).

**Comment by IAmNotHanni** [comment]
+1

**Comment by Croydon** [comment]
@Calinou: 

> Perhaps we could have hitsounds depending on damage (higher pitch = more damage)?

**Comment by syn228** [comment]
Is this still open? I would like to tackle this issue as a first-time contributor.

**Comment by MartinMuzatko** [comment]
@syn228 yep! It is ready to be taken. Make sure to look into the following repository: https://github.com/inexorgame/ui-client-hud
Also this needs to be sent via the inexor tree through the core and flex. That is all I know.

We are constantly improving how things are organized.
However, putting in the first effort makes it easier to port this to any other migration we have.

**Comment by Croydon** [comment]
I can't recommend to take this over right now as a `good first issue`.

There is no sounds output and the UI technology stack is broken and/or incomplete in every single point.
